store/transfer_processor.go

```diff
@@ -270,22 +289,54 @@ func (s *transferProcessor) process(ctx context.Context, events []*model.Transfe
 					totalBalances[sub.Group][token] = new(big.Int).Sub(totalBalances[sub.Group][token], f)
 				}
 			}
+
+			// add miner reward
+			if mr, ok := minerRewardDiff[addr]; ok && token == model.ETHAddress {
```

Comment:
hmmm... i dont think this is correct.
`minerRewardDiff` and `undleRewardDiff` don't have the same addresses as in `addrDiff` so you cannot do this update in the loop over `addrDiff`